tasks.py

```diff
@@ -0,0 +1,200 @@
+"""
+tasks.py defines tasks to run via ssh on one of the defined hosts.
+The functions with the @task decoration can be executed from the cli
+by calling `invoke task arg1 ... argn`, e.g. `invoke backup staging`.
+The caller has to be set up to make ssh connections to the intended host
+as the defined user.
+
+## Defined tasks
+* backup env
+    starting a routine to create an encrypted backup of the database
+    in the environment `env`.
+* deploy env github_user github_token
+    downloading the most recent image and code for the given environment,
+    then migrating the database if any migrations are necessary,
+    finally restarting the docker containers. The github user and token are
+    used to download the docker image from the github containe registry.
+* cleanup env
+    do cleanup tasks. At the moment this consists of removing unused docker
+    to free up space.
+
+
```

Comment:
Nit: I think it'd be worth getting rid of these two empty new lines.

Reply:
Done

Reply:
Yes, we do have pre-commit hooks in place, including the end-of-file-fixer (see https://github.com/lucyparsons/OpenOversight/blob/develop/.pre-commit-config.yaml) and they ran over this PR: https://github.com/lucyparsons/OpenOversight/actions/runs/4458996732/jobs/7831175755
These extra empty lines were in the middle of a file, not at the end